### ISSUE TITLE
Notify about errors and warnings

### ISSUE THREAD

**Issue by bastik-tor** [mbunkus/mkvtoolnix#1196]
Currently the new GUI does not report warnings to the user, which might make him/her not check the file, to find out far later that it is not being played correctly and either blaming the player or the muxing tool.

(It is an assumption that warnings, those that prevent muxing, aren't shown either)

**Comment by mbunkus** [comment]
Partial implementations in [GUI: show number of warnings/errors in the status line](/mbunkus/mkvtoolnix/commit/a6c5f22ba99ceeb9c80cd96b48fefd444331ab86), [GUI: context menu entries for acknowledging warnings and errors](/mbunkus/mkvtoolnix/commit/ded994b34388c3e304983385ecd16472657d67bc).

**Comment by bastik-tor** [comment]
To me it seems fair enough to have the status bar with its warnings and the job queue plus the job output which can be opened for any job now, as far as warnings are concerned.

I am undecided if errors require more notifications. Maybe they deserve a popup, when the job queue is done. Apparently you are either undecided as well or there are already other plans to further notify the user about anything that went wrong or might have went wrong.

**Comment by mbunkus** [comment]
I'm not really undecided. So far my plan was to stick with the notifications in the status bar. If you want to give it a try: [pre-builds 739 and newer](https://www.bunkus.org/videotools/mkvtoolnix/windows/pre/). Note that the icons in the status bar will blink (toggle between grayscale and colored icons each second) as long as there are unacknowledged warnings/errors, and I hope that to be enough notifications. The human eye is usually pretty good in spotting such kinds of motion / changes in an otherwise static scene.

That I haven't closed this issue is mostly due to #1210.

**Comment by bastik-tor** [comment]
Thanks, I already tried before I replied to tis issue to state that I consider myself to be "satisfied" with the solution.

At least I noticed it right away, but I was aware of that change being made to the program so I had to expect that it would at least show a number. (I like the animation of the job output when muxing is started, thanks for that.) So I knew where I had to look for errors and I noticed the blinking right away, which lead me to the conclusion that it is at least enough for the warnings.

Warnings occur, but errors usually don't. I don't think I have any file to produce an error. Well I guess the handling of errors is alright how it will be in the next release. Of course that is just my opinion and others might disagree.
